delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/http/HttpClient.scala

```diff
@@ -47,52 +46,56 @@ trait HttpClient {
 
 object HttpClient {
 
+  type AkkaSource = Source[ByteString, Any]
```

Comment:
This is not only used in the HttpClient

Reply:
I know. It is used anywhere under the sdk. It seems a good place to put it.

Reply:
There is the package object in the sdk ?